is_iot_collector/readings/light_intensity.py

```diff
@@ -30,8 +30,6 @@ def raw_value(self):
         
     def __calculate_percentage(self, raw_value):
         percentage = abs((raw_value - self.low_limit) / (self.high_limit - self.low_limit)) * 100
-        if percentage < 0:
-            percentage = 0
         if percentage > 100:
             percentage = 100
```

Comment:
Considering we have the low limit (176) and the high limit (4000), is it possible for the sensor to read values that are not between those two? If not, the second if can be removed too
@pdany1116 only you this thing :smile:

Reply:
I'll approve it and we can solve it in another PR if it's possible to remove the second if too.